src/main/java/org/intellij/sdk/language/BallerinaAnnotator.java

```diff
@@ -0,0 +1,145 @@
+package org.intellij.sdk.language;
```

Comment:
Lets use the existing package structure we have for the new plugin code as well

```suggestion
package io.ballerina.plugins.idea;
```